README.rst

```diff
@@ -13,6 +13,9 @@ Satpy
 .. image:: https://badge.fury.io/py/satpy.svg
     :target: https://badge.fury.io/py/satpy
 
+.. image:: https://zenodo.org/badge/51397392.svg
+   :target: https://zenodo.org/badge/latestdoi/51397392
```

Comment:
Where did you get these URLs from? Clicking on the badge on the zenodo page shows me:

```restructuredtext
.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.3250583.svg
   :target: https://doi.org/10.5281/zenodo.3250583
```

Reply:
here: https://zenodo.org/account/settings/github/
the badge you have will point only to v0.16.0 iiuc

Reply:
Ah sounds good. That's what it looks like to me too.